src/main/java/Exception/DoneFormatException.java

```diff
@@ -0,0 +1,11 @@
+package Exception;
+
+public class DoneFormatException extends RuntimeException {
+    public DoneFormatException(){};
```

Comment:
As far as possible, it's best to avoid code that is 'dead' or unnecessary, like empty constructors (unless you have multiple constructors!)